table_foosball/table_foosball_01.rb

```diff
@@ -0,0 +1,141 @@
+require "securerandom"
+
+class Player
+  def initialize name
+    @name = name
+    @games = [] # what are all the games this player has been involved in?
+    
+    # IDEAS for attributes
+    # - Time stamp for creation of this player object
+    # - UUID because multiple players might have similar names
+    @timestamp = Time.now
+    @id = ID.get_id
```

Comment:
There's no reason to abstract a UUID into another class. It's very appropriate to use a UUID it here, good job :)